docs/Project-Configuration/Create-a-Project.md

```diff
@@ -6,6 +6,12 @@ This guide is meant to help users create their own projects on Evergreen.
 
 To have access to create a project you must be either a super user or the admin of an existing project.
 
+### Things to consider before creating a project
+1. Does Evergreen support the Github organization (e.g., mongodb, 10gen)? If the organization you need is not in the GitHub Organization drop-down, reach out to the Evergreen team to see if it can be supported. 
```

Comment:
I think this should be under "Steps To Create", otherwise it's not quite clear what dropdown is being specified (it's also only relevant in that case). Could be worth noting explicitly that we don't support forks.